sdk/iot/Azure.Iot.Hub.Service/tests/DevicesClientTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace Azure.Iot.Hub.Service.Tests
+{
+    public class DevicesClientTests : E2eTestBase
+    {
+        public DevicesClientTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [Test]
+        public async Task Devices_Lifecycle()
+        {
+            // TODO: This is just a verification that tests run and it requires the tester to complete this test however they see fit.
+            string testDeviceName = GetRandom();
+
+            IoTHubServiceClient client = GetClient();
+
+            try
```

Comment:
Just to set a pattern, we try to follow the pattern to add a try catch around the whole test and clean up in the end even if test failed mid way. And probably assert if cleanup fails. I know this is just a placeholder but probably adding a skeleton will help others write tests consistently.